lib/react.js

```diff
@@ -277,21 +277,21 @@ declare module react {
     ...
   };
 
-  declare export function forwardRef<Config, Instance>(
+  declare export function forwardRef<Config, Instance, Renders = React$Node>(
```

Comment:
I don't think we need the default here. It can be easily inferred.

Reply:
I kept the default value for all the existing usage of `forwardRef`. Without the default value, all the usages of `forwardRef<Props, Instance>` will break.